.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,37 @@
+name: Commitlint
```

Comment:
Is there a commit linter for ocs operator or other odf repos? We may not have the same checks, but for all the downstream-only repos like this, hopefully prow can give the consistency, so maybe we don't need this commit linter?

Reply:
yeah removed this commit